### PR TITLE
AI Camera update

### PR DESCRIPTION
**Pull request by SakuraTheBabyfur**
This PR replaces the AI "Camera List" with one similar to the security camera monitor console

### CODE REVIEW

**Comment by tigercat2000** [comment]
Oh jesus that commit tree

**Comment by Spacemanspark** [comment]
That's a lot of commits.
As for the PR itself, I'm going with a solid :-1:
The security monitor is obnoxious, and lags down the entire game when I open it (not sure about anyone else).
With the current setup, I can press a single key and find what I'm looking for in seconds without having to ctrl + f and what-have-you.

**Comment by SakuraTheBabyfur** [comment]
ya XD first time using github so i decided to edit the code on the website and then just sync with my server, not realizing... well that happens

File: paradise.dme [hunk 1]
```diff
@@ -1543,6 +1543,7 @@
 #include "code\modules\nano\interaction\zlevel.dm"
 #include "code\modules\nano\modules\alarm_monitor.dm"
 #include "code\modules\nano\modules\atmos_control.dm"
+#include "code\modules\nano\modules\camera_moitor.dm"
```
Comment (Crazylemon64) [inline paradise.dme]: Typo!

**Comment by Crazylemon64** [comment]
https://github.com/ParadiseSS13/Paradise/pull/3225

This thread might have nice advice on how to squash that plethora of commits

**Comment by KasparoVy** [comment]
git rebase -i HEAD~31

'pick' on the top one, 'squash' on all the others

in the file afterward, change the first uncommented line to the name of the new commit

**Comment by Crazylemon64** [comment]
Oh! This is working with the master branch... You should create another branch, so that you can work on more than one feature at once, and so you don't aggregate all changes you've made beforehand...

**Pull request closed without merging**